src/legacy/components/ImportExport.component.js

```diff
@@ -11,7 +11,7 @@ import FileSaver from "file-saver";
 import moment from "moment";
```

Comment:
Please see the comments in https://github.com/EyeSeeTea/user-extended-app/pull/235#issuecomment-1188381596 for the review. Thanks!

Reply:
Fixed importFromJson TypeError and icons.
Merged functions: namesFromCollection & namesArrayFromCollection => namesFromCollection, getPlainJsonUser & getPlainUser => getPlainUser and exportToCsv & exportToJson => exportUsers.
Add csvWarningsToJsonWarning function to adapt the CSV warnings to JSON warnings.

csv warnings format:
`Value not found: ${missingValue} [username=${username || "-"} csv-row=${rowIndex} csv-column=${field}]`

json warnings format:
`Value not found: ${missingValue} [username=${username || "-"} json-key=${field}]`